mantidproject/mantid
# Add a setting for default colormap

Report to: SANS group / Rob Dalgliesh

**Description of work.**
This PR adds an option for the default colormap for image plots to the settings window.

**To test:**
Go to File->Settings and open the Plots tab.
Scroll to the bottom and there should now be a dropdown list for setting the default colormap and a checkbox for reversing the colormap.
Change the setting and make an image plot, e.g. colorfill or surface, and check that it has the colormap you set.

Fixes #28602 

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
